### ISSUE TITLE
The models 'r2plus1d_34_clip32_ft_kinetics_from_ig65m' fine-tuned on UCF101 can't get good performance, just 78%, why?

### ISSUE THREAD

**Issue by fwcheng** [moabitcoin/ig65m-pytorch#28]
Thanks for your models. I met with some problem when using the model 'IG-65M+KINETICS+32X112X112'. I fine-tuned on UCF101 dataset using it, but the result is 78% approximately, which is far away from the results in the paper. Did you get the same results of paper's author? If so, can you share more details about your experiments. Thanks very much!

**Comment by daniel-j-h** [comment]
I don't know your fine-tuning protocol, code, or methodology - so can't say anything about your specific results. The official caffe2 models are here if you want to try them instead:

https://github.com/facebookresearch/vmz

We have a ticket open for replicating the official results on Kinetics (see https://github.com/moabitcoin/ig65m-pytorch/issues/2) but it's currently blocked by Kinetics being impossible to get our hands on.

**Comment by sandhawalia** [comment]
@fwcheng - Can you share your fine-tuning code/repo with us.

**Comment by daniel-j-h** [comment]
@fwcheng any updates here?

**Comment by fwcheng** [comment]
> @fwcheng any updates here?

Sorry for the late reply. I have solved this problem, and the model parameters you provided are useful.

**Comment by fwcheng** [comment]
> @fwcheng - Can you share your fine-tuning code/repo with us.

I haven‘t  finished my project, so I didn't share it on Github.

**Comment by daniel-j-h** [comment]
Great you could solve the problem on your end! Closing here as not actionable on our side.

**Comment by DAVEISHAN** [comment]
Hey @fwcheng nice to hear that you got your problem solved. How much accuracy are you getting on UCF-101 now? Did you make any changes in the model/ pretrained weights to achieve that?

Eagerly waiting for your reply. 

Regards,
Ishan

**Comment by fwcheng** [comment]
> Hey @fwcheng nice to hear that you got your problem solved. How much accuracy are you getting on UCF-101 now? Did you make any changes in the model/ pretrained weights to achieve that?
> 
> Eagerly waiting for your reply.
> 
> Regards,
> Ishan

Sorry for the late reply. The results I got on UCF101 is 96.8%， and I haven't change the pretrained weights, I just changed the learning rate for finetune.

**Comment by Madara321** [comment]
Hi @fwcheng I also got about 80% on UCF101, could you share more details about finetune?

**Comment by FesianXu** [comment]
@fwcheng Hi, to get 96.8% on UCF101, how many clips did you use ?

**Comment by Yueeeeee-1** [comment]
hi, same problem i've met as you met before, the result is only 78.5% when I fine-tuned on UCF101, can you show more details about the learning rate setting, and can you share the way of updating learning rate?  Thank you！

**Comment by paden118** [comment]
> hi, same problem i've met as you met before, the result is only 78.5% when I fine-tuned on UCF101, can you show more details about the learning rate setting, and can you share the way of updating learning rate? Thank you！

Hi, what is the accuracy of your final fine-tuning on ucf101?

**Comment by fwcheng** [comment]
您好，您的邮件我已经收到。

**Comment by paden118** [comment]
你好，请问你是微调所有层吗？学习率是如何设置的呢？比如所有卷积层和最后一层全连接层的学习率是分别设置多少？谢谢
